backend/src/main/java/com/fydp/backend/model/PdfInfo.java

```diff
@@ -11,6 +11,7 @@
     private List<Bookmark> chapters;
     private String pdfText;
     private String fileName;
+    private Long summaryId;
```

Comment:
Is it a one-to-one relationship for a PDF and a summary ID? Is it possible for a PDF to have multiple summary ID's (for each chapter)?

Reply:
its one to one, for multiple summaries we will create multiple summary ids. Look at parseChapters method in the AppController, basically for each chapter we send off a summary and return that chapter title with the corresponding summary_id to frontend. 

We need each chapter to have its own id so we can split it up when sending to kafka otherwise we would have to send all the chapters as one big chunk.

Reply:
Oh, so this field only gets used in the cases where there is 0 or 1 summaries generated for a PDF. And the multiple summary case is covered in the `/upload/chapters` endpoint with `chpId`.